src/client/modules/ExportWins/Review/CookiePage/saga.js

```diff
@@ -0,0 +1,26 @@
+import { put, take } from 'redux-saga/effects'
+import { eventChannel } from 'redux-saga'
+
+const storageChannel = eventChannel((emit) => {
+  window.addEventListener('storage', emit)
+  return () => window.removeEventListener(emit)
+})
+
+/**
+ * This ensures that when the user sets their cookie prefference
```

Comment:
```suggestion
 * This ensures that when the user sets their cookie preference
```
There's a minor typo (although it's in a comment so not sure if it matters)